src/overlays/actors/ovl_Demo_Sa/z_demo_sa.c

```diff
@@ -628,18 +617,14 @@ void func_8098F8F8(DemoSa* this) {
     s32 ff = 0xFF;
     f32* unk_1A0 = &this->unk_1A0;
     f32 temp_f0;
-    s32 temp_f18;
 
     *unk_1A0 += 1.0f;
     temp_f0 = kREG(17) + 10.0f;
 
     if (temp_f0 <= *unk_1A0) {
-        this->unk_1A4 = ff;
-        this->actor.shape.unk_14 = ff;
+        this->actor.shape.unk_14 = this->unk_1A4 = ff;
```

Comment:
Is it possible to get rid of that `ff` temp?
```suggestion
        this->actor.shape.unk_14 = this->unk_1A4 = 0xFF;
```

Reply:
I'll double check, but the `ff` thing was a decomp-permuter find because for some reason it was putting 0xFF into registers multiple times unnecessarily